### DISCUSSION TITLE
No way to ignore activities from specific repo

### DISCUSSION THREAD

**Discussion by Masynchin** [community/community#88704] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I've already pressed "Show less activity like this -> I am not interested in ...", but no result.

**Comment by LucaBlackDragon** [comment]
Same issue here.

Example use case:
I'm interested in John Doe's activity, but not in `some-repo` activities.
If John Doe pushes something to `some-repo`, I keep seeing those in my feed, no matter how many times I click on "I am not interested in `some-repo`"

**Comment by AlinaFreed** [comment]
Hi
